docs/CONFIG-PROPERTIES.md

```diff
@@ -33,6 +33,7 @@
 | timer.appcontainer.stats.interval | integer in seconds | 300 | collect application container stats |
 | timer.vault.ready.cutoff | integer in seconds | 300 | reboot after inaccessible vault |
 | maintenance.mode | "enabled" or "disabled" | "none" | don't run applications etc |
+| newlog.allow.fastupload | "enabled" or "disabled" | disabled | allow faster upload gzip logfiles to controller |
```

Comment:
As I can see [here](https://github.com/naiming-zededa/eve/blob/7bd6221e180871c2515df73be2836ef65a454f8f/pkg/pillar/types/global.go#L729), newlog.allow.fastupload defined as BoolItem, not TriStateItem. Seems, that we need to change the line.

Reply:
With `newlog.allow.fastupload: true` it works well as a point of Eden.
So, we need to accept https://github.com/lf-edge/adam/pull/50 as a next step.

Reply:
> As I can see [here](https://github.com/naiming-zededa/eve/blob/7bd6221e180871c2515df73be2836ef65a454f8f/pkg/pillar/types/global.go#L729), newlog.allow.fastupload defined as BoolItem, not TriStateItem. Seems, that we need to change the line.

@giggsoff fixed, thanks.